etstool.py

```diff
@@ -107,7 +107,7 @@
     'pyqt': {'pyqt<4.12'},  # FIXME: build of 4.12-1 appears to be bad
     # XXX once pyqt5 is available in EDM, we will want it here
     'pyqt5': set(),
-    'wx': {'wxpython'},
+    'wx': {'wxpython<3.0.2.0-6'},
```

Comment:
I'd suggest adding a comment on the reason for the version restriction; that way, people looking at this code in two years' time will have some way to figure out whether the restriction still applies or not.

Reply:
done